### DISCUSSION TITLE
Preparing for a new standard of multi-modal data science, ensuring provenance across imaging, genomics, digital biomarkers, CRF and routinely collected data

### DISCUSSION THREAD

**Discussion by epijim** [rinpharma/positconf-roundtables-2023#3] [Ideas]
Traditionally data science in clinical trials has been applied to small, tabular files in CDISC format and analysed in a single language not used in data science. As new modalities like imaging end-points, digital biomarkers, and RWD enter confirmatory clinical reporting we still traditionally have a hand-over where endpoints are processed in R/Python, and then the processed output is integrated as external data into CDISC formats.

With the move to open source for clinical reporting, it is now possible to introduce integrations and potentially enable imaging/omics/digital processing to happen in the same platform as the clinical reporting.

**Comment by shannonpileggi** [comment]
To me, this touches on the existing electronic data capture infrastucture, and whether it is flexible / adaptable enough to accomodate data from external sources. I would be curious to hear what platforms people are using that allow for these integrations. 

Is this in line with your thinking or something different?

**Comment by brianrepko** [comment]
I work on biomarker research for bed-to-bench (back-translation) - bulk RNA, DNA, scRNA, etc. as well as real-world data (and OMOP / OHDSI models). We keep these separate now but are looking into the CDISC GF domain for some of these as well as integrative analysis combining RWD (or RWD with genomic results) with our clinical trial data.

**Comment by bargop** [comment]
I think there is great opportunity to combine efforts with HL7's Vulcan project for FHIR (https://confluence.hl7.org/display/VA/Vulcan+Accelerator+Home). Many pharma companies are investing in this and defining common ground would be beneficial as we could create tools in open source that address these modern data types. I know a couple of people in Vulcan and can ask them about collaborations.
